setup.py

```diff
@@ -21,8 +22,9 @@
 
 class Unsupported(TestCommand):
     def run(self):
-        print("Running 'test' with setup.py is not supported. "
-              "Use 'pytest' or 'tox' to run the tests.")
+        sys.stderr.write("Running 'test' with setup.py is not supported. "
```

Comment:
Hm, I'm torn - it seems like it would be easier to raise an exception, but that's pretty spammy.